consultation_analyser/consultations/jinja2/consultations/consultations/show.html

```diff
@@ -18,6 +18,32 @@ <h1 class="govuk-heading-m">{{ page_title }}</h1>
 
   {%- from "govuk_frontend_jinja/components/breadcrumbs/macro.html" import govukBreadcrumbs -%}
 
+  {% if all_runs %}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+          <div class="iai-panel">
+            <form>
+              <div class="govuk-form-group">
```

Comment:
```suggestion
              <div class="govuk-form-group govuk-!-margin-bottom-3">
```

Reply:
Thanks - have added that suggestion. Do you think there should be a gap here?
![image](https://github.com/user-attachments/assets/25feb74a-5f0d-45b4-98f8-df65d8a4d704)

Reply:
Scrap that - am removing from that page anyway!

Reply:
Just noticed that they're two separate panels. In which case, yes. And would be good to get the left-alignment matching. I'll do an update for that.

Reply:
Have removed from this page (actually doesn't really make sense to have a dropdown here) - but good to know!